snuba/subscriptions/combined_scheduler_executor.py

```diff
@@ -109,9 +110,7 @@ def get_topic_configuration_for_entity(
     )
 
     return StreamProcessor(
-        tick_consumer,
-        Topic(commit_log_topic.topic_name),
-        factory,
+        tick_consumer, Topic(commit_log_topic.topic_name), factory, ONCE_PER_SECOND
```

Comment:
Out of curiosity, why is this one `ONCE_PER_SECOND`? Was that defined somewhere else already? Based on the other changes I assumed `IMMEDIATE` was the default.

Reply:
You're right, this is actually changing from IMMEDIATE to ONCE_PER_SECOND. This is because committing every offset to Kafka is a bad idea and we should actually throttle here. It's the only functional change in this PR.